recipe/meta.yaml

```diff
@@ -1,5 +1,5 @@
-{% set version = "1.2" %}
-{% set sha256 = "7960e1dfdf20d127e75ead931476f2b5c7556df05b117a73880b22ade17d1abc" %}
+{% set version = "30.1.1" %}
+{% set sha256 = "42375df653ad85e6f5b4b0385138b2dbea1f5d66360783d8625c3e4f97f11f0c" %}
```

Comment:
That's quite the version ~~jump~~ *bump* !

Reply:
They went from 1.2 to 21.6.14 (released on 2021 June 14, so presumably a quasi-calver) to 30.0.0 so ... maybe they decided they didn't want to calver after all?